contracts/TomoTokenSale.sol

```diff
@@ -342,20 +305,22 @@ contract TomoContributorWhitelist is Ownable {
 contract TomoTokenSale is Pausable {
   using SafeMath for uint256;
 
-  TomoCoin public token;
+  TomoCoin token;
   TomoContributorWhitelist whitelist;
+  mapping(address => uint256) public participated;
 
   address public ethFundDepositAddress;
   address public tomoDepositAddress;
 
-  uint256 public tokenCreationCap = 4000000 * 10**18;
-  uint256 public totalSupply;
-  uint256 public fundingStartTime = 1504051200; // 2017-08-30
-  uint256 public fundingPoCEndTime = 1519948800; //1518480000; // 2018-02-13
-  uint256 public fundingEndTime = 1530316800; //1518652800; // 2018-02-15
-  uint256 public minContribution = 0.1 ether;
-  uint256 public maxContribution = 10 ether;
-  uint256 public tokenExchangeRate = 4000;
+  uint256 public constant tokenCreationCap = 4000000 * 10**18;
+  uint256 public totalTokenSold = 0;
+  uint256 public constant fundingStartTime = 1519876800; // 2018/03/01 04:00:00
+  uint256 public constant fundingPoCEndTime = 1519963200; // 2018/03/02 04:00:00
+  uint256 public constant fundingEndTime = 1520136000; // 2018/03/04 04:00:00
+  uint256 public constant minContribution = 0.1 ether;
+  uint256 public constant maxContribution = 10 ether;
+  uint256 public constant tokenExchangeRate = 4000;
+  uint256 public maxCap = maxContribution.mul(tokenExchangeRate);
```

Comment:
constant

Reply:
maxCap is not able to be a constant (build error)